katas/es6/language/arrow-functions/binding.js

```diff
@@ -17,21 +17,21 @@ class LexicallyBound {
 
 describe('Arrow functions have lexical `this`, no dynamic `this`', () => {
   it('bound at definition time, use `=>`', function() {
-    var bound = new LexicallyBound();
-    var fn = bound.getFunction();
+    const bound = new LexicallyBound();
```

Comment:
Since this kata does NOT require knowledge about `const` I am using `var` and actually plan to continue to do so.
In the metadata file you see a field `requiresKnowledgeFrom` which eventually will be used to create a learning guide, through the katas, and then `const` would be taught before it is ever used.
Therefore I am sorry, but this does not fit the current plan :(